num.nix

```diff
@@ -153,4 +209,446 @@ rec {
 
     # TODO: polar, magnitude, phase
   };
+
+  /* truncate :: float -> int
+
+     Truncate a float to an int, rounding towards 0.
+  */
+  # may god have mercy on my soul.
```

Comment:
thanks for keeping this comment

Reply:
just trying to keep a healthy appreciation for cursed hacks :)